packages/cli/src/commands/dev/index.ts

```diff
@@ -48,6 +48,25 @@ const help = () => {
 };
 
 export default async function main(client: Client) {
+  client.output.log(
+    `__VERCEL_DEV_RUNNING = ${process.env.__VERCEL_DEV_RUNNING}`
+  );
```

Comment:
Did you intend to print this for the user?

Reply:
No, that's only for debugging purposes. Newly added test is failing on CI but passing locally.